### DISCUSSION TITLE
How to check if I'm in the home directory

### DISCUSSION THREAD

**Discussion by thomaslevesque** [JanDeDobbeleer/oh-my-posh#5730] [Troubleshoot] [answered]
I'd like to show a folder icon using the `path` segment, but only if I'm not in the home directory (in which case I want to show a "home" symbol)
I can't figure out the template syntax. I tried something like this but it doesn't work (always shows the icon)
```json
{
  ...
  "template": "{{ if ne \"C:\\\\users\\\\myusername\" .Path }}\uE613 {{ end }}{{ path .Path .Location }} ",
}
```
Can anyone help me figure it out?

**Comment by lewis-yeung** [comment]
@thomaslevesque You need to use `.PWD` instead of `.Path` in the template.

**Reply by JanDeDobbeleer** [reply]
@lewis-yeung although matching using `~` and `.Path` should be easier.

**Reply by lewis-yeung** [reply, marked as answer]
@JanDeDobbeleer Using `.PWD` just in case `.Path` contains any text style tags.

**Reply by thomaslevesque** [reply]
Thanks, I'll try it!

**Reply by thomaslevesque** [reply]
It works, thanks!
![image](https://github.com/user-attachments/assets/d5c8b2d8-38a9-4479-b62a-8387dc07b318)
The right incantation was `{{ if ne \"~\" .Path }}`, which is simple enough.

**Reply by thomaslevesque** [reply]
Sorry, I got confused. The one that works is actually `{{ if ne \"~\" .PWD }}`
